pkgs/development/python-modules/uc-micro-py/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, coverage
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "uc-micro-py";
+  version = "1.0.1";
```

https://github.com/tsutsu3/uc.micro-py/blob/main/setup.py#L36

```suggestion
  version = "1.0.1";
  disabled = pythonOlder "3.6";
```